tests/HasTranslatableSlugTest.php

```diff
@@ -302,4 +302,56 @@ public function it_can_update_slug_with_non_unique_names_multiple()
             $this->assertSame($expectedSlug, $model->getTranslation('slug', 'en'));
         }
     }
+
+
+    /** @test */
+    public function it_can_resolve_route_binding()
+    {
+        $model = new TranslatableModel();
```

Comment:
Instead of testing it on this low level, create a test in which you create a Route, set up the binding and check if the resolved model that gets past to the route is the model you expect. That would give me more confidence that everything is working in tandem with Laravel.